utils/admin_console_index.tsx

```diff
@@ -1,12 +1,34 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-
 import FlexSearch from 'flexsearch/dist/flexsearch.es5';
```

Comment:
Type check is failing because it can't find a declaration file for module 'flexsearch/dist/flexsearch.es5'.

I tried to declare by just exporting Index type from declaration file but then typescript is complaining about no constructor found. The ideal solution is to update FlexSearch but as explained [here](https://github.com/mattermost/mattermost-webapp/pull/10162#discussion_r850665517 ), it is in the backlog with no priority.

Please let me know how to proceed on that.

Reply:
To add a shim for the types, you might be able to get away with just defining this as an empty module like https://github.com/mattermost/mattermost-webapp/blob/master/types/external/inobounce.d.ts. If I remember correctly, that will just treat all of its exports as an any, so it should fix any typing issues

Reply:
Great hint, thanks. Type check is fixed now